plugins/GeoIp2/LocationProvider/GeoIp2/Php.php

```diff
@@ -320,7 +320,15 @@ public function getInfo()
     private function getGeoIpInstance($key)
     {
         if (empty($this->readerCache[$key])) {
-            $pathToDb = self::getPathToGeoIpDatabase($this->customDbNames[$key]);
+            if (is_string($this->customDbNames[$key])
+                && strpos( $this->customDbNames[$key], '/') !== false
+                && @file_exists($this->customDbNames[$key])) {
+                // for performance reason we only check for file exists if the string contains a slash and is not only a
+                // filename
+                $pathToDb = $this->customDbNames[$key];
```

Comment:
Wondering if it might make sense to move that logic to `GeoIp2::getPathForGeoIpDatabase`. Not sure if there might be use cases that would need to set an array of custom database paths for one type at any time... Otherwise looks good.

Reply:
good point @sgiehl changed it. tested it and worked for me.